src/client/app/redux/reducers/maps.ts

```diff
@@ -6,7 +6,6 @@ import { MapMetadata, MapsAction, MapState } from '../../types/redux/map';
 import { ActionType } from '../../types/redux/actions';
```

Comment:
For the record, another file that needs changes for newer Redux. It also should get away from thinking of multiple edited maps, etc.

Reply:
Noted for future work